<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueTableItem</class>
 <widget class="QWidget" name="QueueTableItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelFileName">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>4</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>4</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar{text-align: center;
                                     background-color: #FFC4C7D0;
                                     border: 0px solid #FFC4C7D0;
                                     border-radius: 2px;}
QProgressBar::chunk{background-color: #FF407AFF;
                                     border-radius: 2px;}</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelProgressStatus">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPhoneNum">
     <property name="text">
      <string>1/1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="toolBtnFinish">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> QToolButton {
     border: none;
     border-radius:2px;
	 background: transparent;
	 color:#FF505465;
 }
QToolButton:hover
{
	border:none;
	background: transparent;
}
 QToolButton:pressed {
     border: 2px solid #FF407AFF;
     border-radius: 2px;
	 background: transparent;
	 /*background-color: #FF407AFF;*/
	 color:#FFFFFFFF;
 }

 QToolButton:flat {
     border: none; /* no border for a flat push button */
 }

 QToolButton:default {
     border-color: navy; /* make the default button prominent */
 }
</string>
        </property>
        <property name="text">
         <string> 上传完成</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/resource/upload/uploadFinish.png</normaloff>:/resource/upload/uploadFinish.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolBtnReupload">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QToolButton {
     border: none;
     border-radius:2px;
	 background: transparent;
	 color:#FF505465;
 }
QToolButton:hover
{
	border:none;
	background: transparent;
}
 QToolButton:pressed {
     border: 2px solid #FF407AFF;
     border-radius: 2px;
	 background: transparent;
	 /*background-color: #FF407AFF;*/
	 color:#FFFFFFFF;
 }

 QToolButton:flat {
     border: none; /* no border for a flat push button */
 }

 QToolButton:default {
     border-color: navy; /* make the default button prominent */
 }
</string>
        </property>
        <property name="text">
         <string> 重新上传</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/resource/upload/reupload.png</normaloff>:/resource/upload/reupload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolBtnDelete">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QToolButton {
     border: none;
     border-radius:2px;
	 background: transparent;
	 color:#FF505465;
 }
QToolButton:hover
{
	border:none;
	background: transparent;
}
 QToolButton:pressed {
     border: 2px solid #FF407AFF;
     border-radius: 2px;
	 background: transparent;
	 /*background-color: #FF407AFF;*/
	 color:#FFFFFFFF;
 }

 QToolButton:flat {
     border: none; /* no border for a flat push button */
 }

 QToolButton:default {
     border-color: navy; /* make the default button prominent */
 }
</string>
        </property>
        <property name="text">
         <string> 删除</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/resource/upload/uploadDel.png</normaloff>:/resource/upload/uploadDel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolBtnCancel">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QToolButton {
     border: none;
     border-radius:2px;
	 background: transparent;
	 color:#FF505465;
 }
QToolButton:hover
{
	border:none;
	background: transparent;
}
 QToolButton:pressed {
     border: 2px solid #FF407AFF;
     border-radius: 2px;
	 background: transparent;
	 /*background-color: #FF407AFF;*/
	 color:#FFFFFFFF;
 }

 QToolButton:flat {
     border: none; /* no border for a flat push button */
 }

 QToolButton:default {
     border-color: navy; /* make the default button prominent */
 }
</string>
        </property>
        <property name="text">
         <string> 取消上传</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/resource/upload/cancelUpload.png</normaloff>:/resource/upload/cancelUpload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
